### DISCUSSION TITLE
Keep input and output in seperate and specific folders

### DISCUSSION THREAD

**Discussion by Betristor** [macroenergy/DOLPHYN#90] [Ideas]
With ehe expansion of coupled energy sectors, the inputs files and output files are getting more and messy. To keep the inputs in seperate folders like power, hsc and csc could help identify which file is used in which sector. And by subfolders seperating them, we could have identical file names for inputs and even identical column names for specific data, which will bring convenience while adding new sectors. And to keep the output files in seperate folders will have same effect in saving our time developing new sectors.

**Comment by dharik13** [comment]
Sub-folders is a great idea. We should do it.
Also consider harmonizing units across supply chains.
